lua/telescope/command.lua

```diff
@@ -101,4 +102,32 @@ function command.run_command(args)
   end
 end
 
+function command.load_command(cmd,...)
+  local args = {...}
+  local user_opts = {}
+  user_opts['cmd'] = cmd
+  user_opts.opts = {}
+
+  for _,arg in ipairs(args) do
+    if arg:find('=',1) == nil then
+      user_opts['extension_type'] = arg
+    else
+      local param = utils.split_by_pattern(arg,'[^=]+')
+      if param[1] == 'theme' then
+        user_opts['theme'] = param[2]
+      elseif param[2]:find(',',1) ~= nil then
+        if param[1] == 'search' then
+          user_opts.opts[param[1]] = param[2]
+        else
+          user_opts.opts[param[1]] = utils.split_by_pattern(param[2],'[^,]+')
```

Comment:
Why not vim.split?

Reply:
this function useful than `vim.fn.split`  with lua regex

Reply:
There is a `vim.split` which is in `shared.lua` which is exactly that and there is a `vim.gsplit`. See 
https://github.com/neovim/neovim/blob/f75be5e9d510d5369c572cf98e78d9480df3b0bb/runtime/lua/vim/shared.lua#L53-L113

Reply:
Got it. remove custom split function